src/app/conf/2025/components/call-for-proposals.tsx

```diff
@@ -0,0 +1,519 @@
+          <h1 className="typography-h2">Call for Proposals</h1>
+          <p className="mt-6 md:mt-10">
+            Putting on an amazing conference depends on great content, which is
+            where you come in! Join other GraphQL leaders and community members
+            as a presenter by submitting to our Call for Proposals (CFP) and
+            sharing your experience across a wide range of topics. Please click
+            through all of the tabs below before submitting a proposal.
+          </p>
+          <p className="mt-6">
+            For any questions regarding the CFP process, please email{" "}
+            <a
+              href="mailto:cfp@linuxfoundation.org"
+              className="text-neu-800 underline hover:no-underline dark:text-neu-50"
+            >
+              cfp@linuxfoundation.org
+            </a>
+            .
+          </p>
+          <p className="mt-6 typography-body-sm md:mt-10">
+            Please be aware that the Linux Foundation uses Sessionize for CFP
+            submissions. Sessionize is a cloud-based event content management
+            software designed to be intuitive and user-friendly. If you need
+            guidance, please review{" "}
+            <a
+              target="_blank"
+              href="https://sessionize.com/playbook/submit-your-session-for-an-event"
+              className="text-neu-800 underline hover:no-underline dark:text-neu-50"
+            >
+              how to submit your session
+            </a>{" "}
+            for an event to see step-by-step instructions and helpful
+            screenshots.
+          </p>
+          <Button
+            disabled={isDisabled}
+            variant="primary"
+            href="https://sessionize.com/graphqlconf-2025?utm_medium=website&utm_campaign=speaker_section"
+            className={clsx(
+              "mt-6 md:mt-10",
+              isDisabled && "cursor-not-allowed",
+            )}
+          >
+            {buttonText}
+          </Button>
+        </div>
+        {/* todo: this might need to be a grid on mobile */}
+        <article className="flex h-auto flex-col bg-[#C6F267]">
+          <div
+            role="tablist"
+            className="flex divide-sec-dark border-b border-sec-dark *:flex-1 max-md:sr-only md:divide-x"
+          >
+            {tabsInOrder.map((tab, i) => (
+              <TabButton
+                tab={tab}
+                tabIndex={i === 0 ? 0 : -1}
+                activeTab={activeTab}
+                setActiveTab={setActiveTab}
+              />
+            ))}
+          </div>
+          <div className="flex flex-1 justify-center overflow-hidden max-md:flex-col md:items-center">
+            {tabsInOrder.map(tab => (
+              <>
+                <TabButton
+                  tab={tab}
+                  activeTab={activeTab}
+                  setActiveTab={setActiveTab}
+                  className="md:hidden"
+                  aria-hidden
+                />
```

Comment:
tabs are twice in the DOM, once in a tablist for desktop and screen readers, and the second time for mobiles (aria-hidden) to make it look like an accordion